site/public/js/drag-and-drop.js

```diff
@@ -17,6 +17,9 @@ var label_array = [];
 var use_previous = false;
 var changed = false;        // if files from previous submission changed
 
+var total_files_added = 0;
+var MAX_NO_OF_FILES;
```

Comment:
Use `MAX_NUM_OF_FILES` as the name.

Use `let` keyword please for new variables.